test/board_test.exs

```diff
@@ -0,0 +1,66 @@
+defmodule BoardTest do
+  use ExUnit.Case
+
+  test "creates 9 cells" do
+    assert Board.create_cells(3) == [1, 2, 3, 4, 5, 6, 7, 8, 9]
+  end
+
+  @cells Board.create_cells(3)
+  @board_3x3 %Board{size: 3, cells: @cells, marks: [:X, :O]}
```

Comment:
Theser two things can be connected to a common function, maybe `Baord.empty_with_size(3)`?
Or just `Board.empty(3)`?